src/app/helpers/whitelistApis.js

```diff
@@ -205,6 +205,10 @@ const API_LIST = {
     '/learner/data/v1/system/settings/get/:slug': {
       checksNeeded: []
     },
+    '/learner/data/v1/system/settings/get/googleReCaptcha': {
+      checksNeeded: ['ROLE_CHECK'],
```

Comment:
is the person who will use this will be loggedin if not then the ROLE_CHECK will not work

Reply:
Yes, the user is logged in user